bom/build.gradle

```diff
@@ -0,0 +1,105 @@
+description = 'gRPC: BOM'
+
+buildscript {
+  repositories {
+    mavenLocal()
+    maven {
+      url 'https://plugins.gradle.org/m2/'
+    }
+    mavenCentral()
+    jcenter()
+  }
+}
+
+archivesBaseName = "bom"
+
+project.version = "1.19.0-SNAPSHOT" // CURRENT_GRPC_VERSION
+
+apply plugin: 'maven-publish'
```

Comment:
:-/ Now we have both publishing mechanisms. That's not great, and would lead to lots of confusion. Does this mean we should begin the migration to maven-publish (which would be good)?